.changeset/eight-ducks-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-pagerduty': patch
```

Comment:
And because it is, I think we should highlight that this is a breaking change similar to this format: https://github.com/backstage/backstage/blob/master/docs/releases/v1.2.0-changelog.md#minor-changes-17 using the `**BREAKING**` prefix to the note

Reply:
sounds good, I will update this 👍🏻